src/main/java/com/app/bookstore/service/book/BookService.java

```diff
@@ -10,4 +10,8 @@ public interface BookService {
     BookDto findById(Long id);
 
     List<BookDto> findAll();
+
+    void deleteBookById(Long id);
+
+    BookDto updateBookById(Long id, CreateBookRequestDto createBookRequestDto);

```

Comment:
We can simplify variable name here.
```suggestion
    BookDto updateBookById(Long id, CreateBookRequestDto bookDto);
```